recipes/cargo-insta/build.bat

```diff
@@ -0,0 +1,14 @@
+:: check licenses
+cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+set CARGO_PROFILE_RELEASE_STRIP=symbols  
```

Comment:
```suggestion
set CARGO_PROFILE_RELEASE_STRIP=symbols
```

Reply:
That's a good to one know about!

We may want to see if we can set these environment variables when the `rust` feedstock is activated to avoid these kinds of problems in the future.

Reply:
I proposed adding them to the activation scripts but there was a bit of resistance as these should only be set for executables.